Assets/TabMasterScript.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using Microsoft.MixedReality.Toolkit.UI;
+using UnityEngine;
+
+public class TabMasterScript : MonoBehaviour
+{
+    public tab1 tab_one;
+    public tab2 tab_two;
+    public tab3 tab_three;
+    public tab4 tab_four;
+
+    PressableButtonHoloLens2 button_tab1;
+    Interactable interactable_tab1;
+
+    PressableButtonHoloLens2 button_tab2;
+    Interactable interactable_tab2;
+
+    PressableButtonHoloLens2 button_tab3;
+    Interactable interactable_tab3;
+
+    PressableButtonHoloLens2 button_tab4;
+    Interactable interactable_tab4;
+
+    int active;
```

Comment:
I think I know what this variable does but maybe you want to rename it? Or instead just store a reference to a `Tab curTab`